Gemfile

```diff
@@ -35,6 +35,7 @@ gem 'carrierwave-aws'
 gem 'clam_scan'
 gem 'config'
 gem 'connect_vbms', git: 'https://github.com/department-of-veterans-affairs/connect_vbms.git', branch: 'master'
+gem 'danger'
```

Comment:
Could the dependency go in test/development or is it needed in all environments for the build process?